test_conformance/images/kernel_image_methods/test_1D_buffer.cpp

```diff
@@ -67,9 +67,7 @@ static int test_get_1Dimage_buffer_info_single(cl_context context,
     if (gDebugTrace)
         log_info(" - Creating 1D image %d ...\n", (int)imageInfo->width);
 
-    buffer = clCreateBuffer(
-        context, flags, imageInfo->width * get_pixel_size(imageInfo->format),
```

Comment:
This seems to be the correct calculation but [line 264](https://github.com/KhronosGroup/OpenCL-CTS/pull/2073/files#diff-5c70295908934db772f80e042240dd21b82aae203fab100a992812da85ef3d71L264) in this file modifies this value as such:
```
do
{
    extraWidth++;
    imageInfo.rowPitch += extraWidth;
} while ((imageInfo.rowPitch % pixelSize) != 0);
```